manifests/params.pp

```diff
@@ -39,15 +39,23 @@
           $plugin_config_dir           = '/etc/collectd/managed_config'
           $collectd_config_file        = '/etc/collectd/collectd.conf'
           $filtering_config_file       = '/etc/collectd/filtering_config/filtering.conf'
-          $mysql_socket_file           = '/var/lib/mysqld/mysqld.sock'
+          $bundled_plugins             = [
+                                            'collectd-apache',
+                                            'collectd-kafka',
```

Comment:
docker, mongo and zookeeper are all managed_package = false since they are being installed from the in module source right now so do not need to be in the bundled plugin list for debian based systems. collectd-java and collectd-cassandra should be the only ones that where missed. i've added them and removed the $modules line from java. latest commit is cf66123